spec/controllers/linked_data_terms_controller_spec.rb

```diff
@@ -307,9 +283,27 @@
           stub_request(:get, 'http://id.worldcat.org/fast/530369')
             .to_return(status: 200, body: webmock_fixture('lod_oclc_term_found.rdf.xml'), headers: { 'Content-Type' => 'application/rdf+xml' })
         end
-        it 'succeeds' do
+        it 'succeeds and defaults to json content type' do
           get :show, params: { id: '530369', vocab: 'OCLC_FAST' }
           expect(response).to be_successful
+          expect(response.content_type).to eq 'application/json'
+        end
+
+        context 'and it was requested as json' do
+          it 'succeeds and returns term data as json content type' do
+            get :show, params: { id: '530369', vocab: 'OCLC_FAST', format: 'json' }
+            expect(response).to be_successful
+            expect(response.content_type).to eq 'application/json'
+          end
+        end
+
+        context 'and it was requested as jsonld' do
+          it 'succeeds and returns term data as jsonld content type' do
+            get :show, params: { id: '530369', vocab: 'OCLC_FAST', format: 'jsonld' }
+            expect(response).to be_successful
+            expect(response.content_type).to eq 'application/ld+json'
+            expect(JSON.parse(response.body).keys).to match_array ["@context", "@graph"]
+          end
```

Comment:
These tests were covered using the Agrovoc auth.  They were moved to the oclc_fast tests to maintain coverage of that functionality.